assets/rich_text_components_definitions.js

```diff
@@ -46,8 +46,8 @@ var richTextComponents = {
     "frontend_id": "image",
     "tooltip": "Insert image",
     "icon_data_url": "/rich_text_components/Image/Image.png",
-    "preview_url_template_dev": "/imagehandler/<[explorationId]>/<[filepath]>",
-    "preview_url_template_prod": "https://storage.googleapis.com/<[bucketName]>/<[explorationId]>/assets/image/<[filepath]>",
+    "preview_url_template_dev": "/imagehandler/<[explorationId]>/<[filepath.filename]>",
+    "preview_url_template_prod": "https://storage.googleapis.com/<[bucketName]>/<[explorationId]>/assets/image/<[filepath.filename]>",
```

Comment:
We have the it as filepath in the backend. So that is why I kept it as filepath

Reply:
I think you should rename it to fileinfo if it's going to contain stuff other than the filename.

An exploration migration might be needed. You can talk to @bansalnitish about how this works.

Reply:
Hi @seanlip .Shall I do that in another PR?

Reply:
Isn't this the PR that does an exploration migration? So shouldn't the name change happen there too?

Reply:
Okay Sure.

Reply:
We would not alter this function later on. Actually I meant to have another vn_to_vn+1 function for changing the name from filepath to fileinfo.
Doing that in a separate PR would be clean because it will involve a lot of changes in the frontend and the backend tests too.
Let me know what you think.